modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-text/src/main/java/com/liferay/dynamic/data/mapping/type/text/internal/TextDDMFormFieldTypeSettings.java

```diff
@@ -81,12 +83,17 @@
 public interface TextDDMFormFieldTypeSettings
 	extends DefaultDDMFormFieldTypeSettings {
 
+	@DDMFormField(
+		label = "%autocompleteEnabled", properties = {"showAsSwitcher=true"}
+	)
+	public boolean autocompleteEnabled();
+
 	@DDMFormField(
 		label = "%create-list",
 		optionLabels = {"%manually", "%from-data-provider"},
 		optionValues = {"manual", "data-provider"},
 		properties = {"showLabel=false"}, type = "radio",
-		visibilityExpression = "TRUE"
+		visibilityExpression = "autocompleteEnabled"
```

Comment:
Hey @linolaoj Could you please use `@DDMFormRule` annotation instead of the deprecated `visibilitExpression` property?

Thx